models/articles.models.js

```diff
@@ -0,0 +1,12 @@
+const connection = require("../db/connection");
+
+exports.selectArticleById = ({ article_Id }) => {
+	return connection
+		.query("SELECT * FROM articles WHERE article_id = $1", [article_Id])
+		.then(({ rows }) => {
+			if (rows.length === 0) {
+				return Promise.reject({ status: 400, msg: "Wrong Input" });
+			}
+			return rows;
```

Comment:
You could remove the article from the array here, so a single object is returned rather than a single object in an array